config/api.yml

```diff
@@ -1068,6 +1068,56 @@
     - :collection
     :verbs: *g
     :klass: ChargebackRateDetailMeasure
+  :middleware_datasources:
+    :description: Middleware Datasources
+    :klass: MiddlewareDatasource
+    :options:
+      - :collection
+    :verbs: *g
```

Comment:
It is a reference to this:
https://github.com/ManageIQ/manageiq/blob/master/config/api.yml#L20